<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="ip_label">
       <property name="text">
        <string>IP：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ip_LineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="superButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>超级模式！</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/actions/icons/super.png</normaloff>:/actions/icons/super.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>清除消息显示</string>
       </property>
       <property name="text">
        <string>清屏</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="toolTip">
        <string>按Ctrl+Enter键发送消息</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connect"/>
   <addaction name="action_sendFile"/>
   <addaction name="action_history"/>
   <addaction name="action_quit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_history"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_super"/>
    <addaction name="action_sendFile"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_connect">
   <property name="icon">
    <iconset resource="../newServer/images.qrc">
     <normaloff>:/actions/icons/connect.png</normaloff>:/actions/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>与服务器建立连接</string>
   </property>
  </action>
  <action name="action_sendFile">
   <property name="icon">
    <iconset resource="../newServer/images.qrc">
     <normaloff>:/actions/icons/sendfile.png</normaloff>:/actions/icons/sendfile.png</iconset>
   </property>
   <property name="text">
    <string>传文件</string>
   </property>
   <property name="toolTip">
    <string>传送文件，需退出超级模式</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../newServer/images.qrc">
     <normaloff>:/actions/icons/about.png</normaloff>:/actions/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于程序</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../newServer/images.qrc">
     <normaloff>:/actions/icons/quit.png</normaloff>:/actions/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_super">
   <property name="icon">
    <iconset>
     <normaloff>:/actions/icons/super.png</normaloff>:/actions/icons/super.png</iconset>
   </property>
   <property name="text">
    <string>超级模式！</string>
   </property>
   <property name="toolTip">
    <string>超级模式</string>
   </property>
  </action>
  <action name="action_history">
   <property name="icon">
    <iconset resource="../newServer/images.qrc">
     <normaloff>:/actions/icons/history.png</normaloff>:/actions/icons/history.png</iconset>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
   <property name="toolTip">
    <string>显示聊天记录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
